src/mjolnir/shortcutbuilder.cc

```diff
@@ -423,11 +423,10 @@ uint32_t AddShortcutEdges(GraphReader& reader,
         std::reverse(shape.begin(), shape.end());
       }
 
-      // Get names - they apply over all edges of the shortcut
-      auto names = tile->GetNames(directededge->edgeinfo_offset());
-      auto tagged_names = tile->GetNames(directededge->edgeinfo_offset(), true);
-
-      auto types = tile->GetTypes(directededge->edgeinfo_offset());
+      // Get names and types - they apply over all edges of the shortcut
+      auto names = edgeinfo.GetNames();
+      auto tagged_names = edgeinfo.GetNames();
```

Comment:
Is this missing the bool for tagged names?

Reply:
thanks for the catch! My bad